CONTRIBUTING.md

```diff
@@ -1,29 +1,39 @@
+If you'd like to contribute, but don't have a project in mind, look at the [open issues](https://github.com/awsdocs/aws-doc-sdk-examples/issues) in this repository for some ideas. Any issues with the [help wanted](https://github.com/awsdocs/aws-doc-sdk-examples/labels/help%20wanted) or [enhancement](https://github.com/awsdocs/aws-doc-sdk-examples/labels/enhancement) labels are a great place to start.
 
-The AWS documentation source files on GitHub aren't published directly to the official documentation website. If we merge your pull request, we'll publish your changes to the documentation website as soon as we can, but they won't appear immediately or automatically.
+In addition to written content, we really appreciate new examples for our documentation, such as examples for different platforms or environments, and examples in additional programming languages.
 
-We look forward to receiving your pull requests for:
+Additionally, we look forward to receiving your issues for:
 
 * New content you'd like to contribute (such as new code examples or tutorials) - for more information, see [Types of examples](https://github.com/awsdocs/aws-doc-sdk-examples/edit/main/CONTRIBUTING.md#types-of-examples) below.
 * Inaccuracies in the content
 * Information gaps in the content that need more detail to be complete
 * Typos or grammatical errors
 * Suggested rewrites that improve clarity and reduce confusion
 
-**Note:** We all write differently, and you might not like how we've written or organized something currently. We want that feedback. But please be sure that your request for a rewrite is supported by the previous criteria. If it isn't, we might decline to merge it.
+## What to expect from AWS
 
+When you submit a pull request, our team is notified and will respond as quickly as we can. We'll do our best to work with you to ensure that your pull request adheres to our style and standards. If we merge your pull request, we might make additional edits later for style or clarity.
+
+The AWS documentation source files on GitHub aren't published directly to the official documentation website. If we merge your pull request, we'll publish your changes to the documentation website as soon as we can, but they won't appear immediately or automatically.
+
+
+**Note:** We all write differently, and you might not like how we've written or organized something currently. We want that feedback. But please be sure that your request for a rewrite is supported by the previous criteria. If it isn't, we might decline to merge it.
 
-## Where to put a new code example
+## Write code and submit pull requests
 
 Code examples are organized by service for each SDK. Within each SDK, the folder and example structure varies. To add
```

Comment:
This section, as far down as 'One of the AWS SDK code example team members will follow up with you in that issue.' are 'Guidelines for writing code examples' and could possibly be moved to the top of the 'Types of examples' section.